### PR TITLE
Swith EAPI 5 to EAPI 8

### PR DESCRIPTION
**Pull request by Anard**
I didn't touch any of other ebuild (which are on EAPI 6 or newer).
Repoman still see a problem, but on an ebuild I didn't touch : 
```
RepoMan scours the neighborhood...
  ebuild.minorsyn               1
   net-print/brother-hl3152cdw-bin/brother-hl3152cdw-bin-1.1.4.ebuild: line 13: Ebuild contains leading spaces

Note: use --include-dev (-d) to check dependencies for 'dev' profiles

RepoMan sez: "You're only giving me a partial QA payment?
              I'll take it this time, but I'm not happy."
```
Not tested as I haven't these printers but it seems to be OK.

### CODE REVIEW

**Comment by Anard** [comment]
I just finished to convert all ebuilds of media-gfx and net-print in my branch [allEAPI8](https://github.com/Anard/brother-overlay/tree/allEAPI8)
Still need to check if it's all OK.
Some volunteers ? :)

File: net-print/brother-dcp9055cdn-bin/brother-dcp9055cdn-bin-1.1.1.ebuild [hunk 1]
```diff
@@ -1,68 +0,0 @@
-# Copyright 1999-2016 Gentoo Foundation
```
Comment (stefan-langenmaier) [inline net-print/brother-dcp9055cdn-bin/brother-dcp9055cdn-bin-1.1.1.ebuild:1]: Why was this file deleted/not updated?

File: net-print/brother-mfc9340cdw-bin/brother-mfc9340cdw-bin-1.1.2.ebuild [hunk 2]
```diff
@@ -1,47 +0,0 @@
-# Copyright 1999-2016 Gentoo Foundation
```
Comment (stefan-langenmaier) [inline net-print/brother-mfc9340cdw-bin/brother-mfc9340cdw-bin-1.1.2.ebuild:1]: Why was this file deleted?

**Comment by stefan-langenmaier** [comment]
Thanks for your MR! Looks good to me. I'll have a look regarding the failing RepoMan check.

There are two ebuild that have been removed. Could you elaborate why that is?

**Comment by Anard** [comment]
Hi. I deleted some files when a driver version was duplicated : here is your original repository : 
```$ ls net-print/brother-dcp9055cdn-bin/
brother-dcp9055cdn-bin-1.1.1.ebuild     brother-dcp9055cdn-bin-1.1.1-r4.ebuild
brother-dcp9055cdn-bin-1.1.1-r1.ebuild  Manifest
brother-dcp9055cdn-bin-1.1.1-r2.ebuild  metadata.xml
brother-dcp9055cdn-bin-1.1.1-r3.ebuild
$ ls net-print/brother-mfc9340cdw-bin/
brother-mfc9340cdw-bin-1.1.2.ebuild     files
brother-mfc9340cdw-bin-1.1.2-r1.ebuild  Manifest
brother-mfc9340cdw-bin-1.1.2-r2.ebuild  metadata.xml
```
In these rare cases, I only kept the last revision. Do you want me to reinstate these files ? 
Have you taken a look to allEAPI8 branch ? There are not much differences and we would save time on next portage evolutions if all ebuilds of the repo are converted to EAPI 8. I only have to merge this branch to master to integrate it to PR.
About Repoman, it complains about spaces on 2d line of SRC_URI line. It isn't an error and ebuild is more readable like this.

File: net-print/brother-mfc9340cdw-bin/brother-mfc9340cdw-bin-1.1.2.ebuild (hunk 2 above)
Comment (Anard) [reply net-print/brother-mfc9340cdw-bin/brother-mfc9340cdw-bin-1.1.2.ebuild:1]: After updating an ebuild, we have to change its name. I added -r1 to ebuilds like brother-xxx1234-bin-1.0.ebuild and incremented those like brother-xxx1234-bin-1.0-r1.ebuild.
When multiple r1 r2 ebuilds were found, I kept only the last one and deleted older ones.

**Comment by stefan-langenmaier** [comment]
Thanks for the MR and the explanation. It's good to have the old revisions removed.

**Comment by stefan-langenmaier** [comment]
I also had a look regarding your other branch. I've forgotten how many files there are. :) That must have been a lot of work as I have seen some files with manual modifications as well. There is the `/generator` directory for tools/script helping with the creation of the ebuilds. Maybe you could add your scripts for this rewrite there as well. I'm sure that could help someone else in there future.

So if you can create a MR with your allEAPI8 branch I would like to add it.

Thanks again!
